src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -852,3 +863,12 @@ export const wantsHelpRequestingStatementsSecondary = index => formData =>
   ) &&
   isAnswering781aQuestions(index)(formData) &&
   wantsHelpWithOtherSourcesSecondary(index)(formData);
+
+export const setDefaultAttachmentId = attachmentId => {
```

Comment:
I feel like this needs a better name 🤔

Reply:
`setUploadDocType`? 🤷‍♂️

Reply:
It's not default in some cases as much as the _only_ option, right?

Reply:
As of now that's the case, but only because the `uiSchema` is disabling the select, you could technically reuse this to just set a default.

Reply:
I think it's more of a `getAttachmentsSchema`, myself. `setSomething` implies that there's setting happening on an existing data structure.

The param name can be `defaultAttachmentId`. That seems to make sense if we can override it manually by selecting an option from the dropdown.

Reply:
Yeah I was actually confused by what was going on, but this makes sense on second pass. I think something along the lines of `createUploadSchema` is closest to what's actually happening, but it's not hugely different from `get` so I don't think any names need to change.